src/components/organisms/sections/Hero/index.jsx

```diff
@@ -29,6 +29,7 @@ import "./hero.scss";
 
 //> Settings
 import radarSettings from "./radarSettings.js";
+import radarSettingsDark from "./radarSettingsDark.js";
```

Comment:
The file extensions here aren't required, are they?
```suggestion
import radarSettingsDark from "./radarSettingsDark";
```